client/my-sites/add-ons/hooks/use-add-ons.ts

```diff
@@ -29,18 +29,25 @@ import unlimitedThemesIcon from '../icons/unlimited-themes';
 import isStorageAddonEnabled from '../is-storage-addon-enabled';
 import useAddOnDisplayCost from './use-add-on-display-cost';
 import useAddOnFeatureSlugs from './use-add-on-feature-slugs';
+import useAddOnPrices from './use-add-on-prices';
 
 export interface AddOnMeta {
```

Comment:
Now that AddOnMeta has been defined in `packages/data-store/add-ons/types`, we need to remove this to avoid duplicated definition.

Reply:
[6cdbb1a](https://github.com/Automattic/wp-calypso/pull/81338/commits/6cdbb1a08884ff94e3fef14820eb64f2f311bfa3)